Formula/solr.rb

```diff
@@ -61,6 +50,10 @@ def plist
   end
 
   test do
-    system bin/"solr"
+    # Start a Solr node => exit code 0
+    shell_output(bin/"solr start")
+    sleep 10
```

Comment:
Could you make a query to solr here?

Reply:
yes, already added the queries in previous commit but removed them temporarily

Reply:
:+1: ignore this then